src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -34,14 +34,30 @@ ReportCmcStatus::getPropertyTree20202( const xrt_core::device * _pDevice,
                                            boost::property_tree::ptree &_pt) const
 {
   boost::property_tree::ptree pt;
+  pt.put("Description", "CMC");
   try {
-    pt.put("Description", "CMC status");
     pt.put("heartbeat_err_time", xrt_core::device_query<xrt_core::query::heartbeat_err_time>(_pDevice));
     pt.put("heartbeat_count", xrt_core::device_query<xrt_core::query::heartbeat_count>(_pDevice));
     pt.put("heartbeat_err_code", xrt_core::device_query<xrt_core::query::heartbeat_err_code>(_pDevice));
     pt.put("heartbeat_stall", xrt_core::device_query<xrt_core::query::heartbeat_stall>(_pDevice));
     pt.put("status", xrt_core::utils::parse_cmc_status(static_cast<unsigned int>(xrt_core::device_query<xrt_core::query::heartbeat_err_code>(_pDevice))));
-  } catch(const xrt_core::query::no_such_key&) {}
+  }
+  catch(const xrt_core::query::no_such_key&) {}
+  catch(const xrt_core::query::sysfs_error&) {}

```

Comment:
```suggestion
  catch(const xrt_core::query::exception&) {}
```
Soren introduced this new exception which catches numerous key related issues

Reply:
Hi @AShivangi  the error is related to sysfs node absence. So, this exception will not catch this error. sysfs_error is the right catch.